integration-job/vm/install_deps_and_run_kyma.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
I think that script should be moved to `kyma` repository. `test-infra` should contains only script for provisioning VM.

Reply:
Created a follow-up task: #60